<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonConnect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDisconnect">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGauche">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDroite">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>70</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Droite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAvancer">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Avancer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReculer">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Reculer</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdBatterie">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Hautcam">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Haut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Gauchecam">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bascam">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Bas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Droitecam">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>230</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Droite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>230</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdVitesse">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdInfraGauche">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdInfraHaut">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdInfraDroite">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdInfraBas">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Batterie :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>21</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitesse :</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdPosR">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdPosL">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
